examples/examples.htm

```diff
@@ -100,7 +102,7 @@
             <li><a href='#' id='goto-updating' class='pill'>Updating</a></li>
             <li><a href='#' id='goto-other' class='pill'>Other</a></li>
             <li><a href='#' id='goto-experimental' class='pill'>Experimental</a></li>
-            <li><a href='#' id='goto-addons' class='pill'>Addons</a></li>
+            <li><a href='#' id='goto-brushing_zooming' class='pill'>Brushing & Zooming</a></li>
```

Comment:
Could you put this before `experimental`? Also, you should generally use `&amp;` for specifying the ampersand in html :)